mockkubeapiserver/kubernetes_builtin_schema.yaml

```diff
@@ -0,0 +1,12172 @@
+  map:
+    fields:
+    - name: apiVersion
+      type:
+        scalar: string
+    - name: deleteOptions
+      type:
+        namedType: io.k8s.apimachinery.pkg.apis.meta.v1.DeleteOptions
+    - name: kind
+      type:
+        scalar: string
+    - name: metadata
+      type:
+        namedType: io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta
+      default: {}
+- name: io.k8s.api.policy.v1.PodDisruptionBudget
+  map:
+    fields:
+    - name: apiVersion
+      type:
+        scalar: string
+    - name: kind
+      type:
+        scalar: string
+    - name: metadata
+      type:
+        namedType: io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta
+      default: {}
+    - name: spec
+      type:
+        namedType: io.k8s.api.policy.v1.PodDisruptionBudgetSpec
+      default: {}
+    - name: status
+      type:
+        namedType: io.k8s.api.policy.v1.PodDisruptionBudgetStatus
+      default: {}
+- name: io.k8s.api.policy.v1.PodDisruptionBudgetSpec
+  map:
+    fields:
+    - name: maxUnavailable
+      type:
+        namedType: io.k8s.apimachinery.pkg.util.intstr.IntOrString
+    - name: minAvailable
+      type:
+        namedType: io.k8s.apimachinery.pkg.util.intstr.IntOrString
+    - name: selector
+      type:
+        namedType: io.k8s.apimachinery.pkg.apis.meta.v1.LabelSelector
+- name: io.k8s.api.policy.v1.PodDisruptionBudgetStatus
+  map:
+    fields:
+    - name: conditions
+      type:
+        list:
+          elementType:
+            namedType: io.k8s.apimachinery.pkg.apis.meta.v1.Condition
+          elementRelationship: associative
+          keys:
+          - type
+    - name: currentHealthy
+      type:
+        scalar: numeric
+      default: 0
+    - name: desiredHealthy
+      type:
+        scalar: numeric
+      default: 0
+    - name: disruptedPods
+      type:
+        map:
+          elementType:
+            namedType: io.k8s.apimachinery.pkg.apis.meta.v1.Time
+    - name: disruptionsAllowed
+      type:
+        scalar: numeric
+      default: 0
+    - name: expectedPods
+      type:
+        scalar: numeric
+      default: 0
+    - name: observedGeneration
+      type:
+        scalar: numeric
+- name: io.k8s.api.policy.v1beta1.AllowedCSIDriver
+  map:
+    fields:
+    - name: name
+      type:
+        scalar: string
+      default: ""
+- name: io.k8s.api.policy.v1beta1.AllowedFlexVolume
+  map:
+    fields:
+    - name: driver
+      type:
+        scalar: string
+      default: ""
+- name: io.k8s.api.policy.v1beta1.AllowedHostPath
+  map:
+    fields:
+    - name: pathPrefix
+      type:
+        scalar: string
+    - name: readOnly
+      type:
+        scalar: boolean
+- name: io.k8s.api.policy.v1beta1.Eviction
+  map:
+    fields:
+    - name: apiVersion
+      type:
+        scalar: string
+    - name: deleteOptions
+      type:
+        namedType: io.k8s.apimachinery.pkg.apis.meta.v1.DeleteOptions
+    - name: kind
+      type:
+        scalar: string
+    - name: metadata
+      t
```

Comment:
That's the type we use for objects that don't have a schema (i.e. CRDs with no schema for example).